### PR TITLE
Nexus-8905 Bookmarks FT

### PR DESCRIPTION
**Pull request by kellyrob99**
https://issues.sonatype.org/browse/NEXUS-8905

### CODE REVIEW

File: testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/framework/bookmarking_authenticated.t.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests bookmarking as an admin.
+ */
+StartTest(function(t) {
+
+  var ignoredFeatures = [
```
Comment (kellyrob99) [inline testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/framework/bookmarking_authenticated.t.js]: These features all have some issues, partially for unimplemented functionality, partly for timing related issues that cause exceptions when navigating quickly between pages.
I'll add Jiras for things that definitely need fixing.

File: testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js [hunk 2]
```diff
@@ -153,7 +153,10 @@ Class('NX.TestClass', {
         },
         { type: 'admin[TAB]', target: this.cq1('#username') },
         { type: 'admin123', target: this.cq1('#password') },
-        { type: '[ENTER]' }
+        { type: '[ENTER]' },
+        function(next) {
+          this.waitForAnimations(next);
+        }
```
Comment (mcculls) [inline testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js]: Assuming this is the canonical source, can we now remove the duplicate copies of TestClass.js and TestHarness.js from "testsuite/functional-testsuite/src/test/ft-resources/testsuite-lib/NX" under each of the Nexus repos?

**Comment by mcculls** [comment]
+1

File: testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js (hunk 2 above)
Comment (jdillon) [inline testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js]: @mcculls those are removed, but are overlayed over src for NEXUS_RESOURCE_DIRS to function, but are marked with .gitignore

Comment (mcculls) [inline testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js]: Righto, thanks for the explanation

Comment (kellyrob99) [inline testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js]: FTR I will be going through and removing any extraneous calls in existing code shortly

**Comment by kellyrob99** [comment]
+1 by CI http://bamboo.s/browse/NX3-OSSF524-TEST-5

**Pull request closed without merging**